themes/jquery/header.php

```diff
@@ -104,6 +104,11 @@
 	</section>
 </header>
 
+<div id="banner-secondary">
```

Comment:
This is an existing ID since there's already some CSS for it. jqueryui.com relies on it, for example. Modifying styles for that one may be tricky. It might be safer to create a new id or a class and style it from scratch.

Reply:
OK to add a new class to base.css, perhaps in the nav section?

Reply:
added banner-blm to section with other global banners